components/footer.tsx

```diff
@@ -16,13 +16,14 @@ const Footer = () => {
         <div className="flex flex-col justify-between gap-x-2 gap-y-8 px-8 md:flex-row">
           <div className="flex flex-col md:gap-2">
             <Link href="/" className="w-fit">
-              <Image
-                src={"/codedevils-logo.svg"}
-                alt="CodeDevils logo"
-                width={150}
-                height={150}
-                className=""
-              />
+            <div className="w-[150px] h-[150px] relative">
```

Comment:
I see the approach you're taking to solve the issue, but the additional div wrapper isn't necessary and introduces a inefficiency. It would be more effective to apply the styles directly to the parent `Link` tag instead if you wanted to go this route.